### PR TITLE
Fee register ap iand frontend

### PR DESCRIPTION
**Pull request by anshikanigam15**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: apps/bar/bar-api/demo-int-image-policy.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: demo-int-bar-api
+  annotations:
+    hmcts.github.com/prod-automated: disabled
+spec:
```
Comment (thomast1906) [inline apps/bar/bar-api/demo-int-image-policy.yaml:7]: needs added to autmation/kustomization.yaml of the namespace

File: apps/bar/bar-web/demo-int-image-policy.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: demo-int-bar-web
+  annotations:
+    hmcts.github.com/prod-automated: disabled
```
Comment (thomast1906) [inline apps/bar/bar-web/demo-int-image-policy.yaml:6]: needs added to autmation/kustomization.yaml of the namespace

File: k8s/namespaces/bar/bar-api-int/bar-api-int.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+spec:
+  releaseName: bar-api-int
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/bar-api
+  values:
+    java:
+      replicas: 2
+      image: hmctspublic.azurecr.io/bar/api:prod-9b64ade-20220811171838 #{"$imagepolicy": "flux-system:bar-api"}
```
Comment (thomast1906) [inline k8s/namespaces/bar/bar-api-int/bar-api-int.yaml:14]: Does image not need updated with new imagePolicy being added?

File: k8s/namespaces/bar/bar-api-int/demo.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: bar-api-int
+spec:
+  releaseName: bar-api-int
+  values:
+    java:
+      image: hmctspublic.azurecr.io/bar/api:prod-9b64ade-20220811171838 #{"$imagepolicy": "flux-system:bar-api"}
```
Comment (thomast1906) [inline k8s/namespaces/bar/bar-api-int/demo.yaml:9]: Does image not need updated with new imagePolicy being added?

File: k8s/namespaces/bar/bar-web-int/bar-web-int.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/bar-web
+  values:
+    nodejs:
+      replicas: 2
+      image: hmctspublic.azurecr.io/bar/web:prod-9dcca15-20220808172206 #{"$imagepolicy": "flux-system:bar-web"}
```
Comment (thomast1906) [inline k8s/namespaces/bar/bar-web-int/bar-web-int.yaml:17]: Does image not need updated with new imagePolicy being added?

File: k8s/namespaces/bar/bar-web-int/demo.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: bar-web-int
+spec:
+  releaseName: bar-web-int
+  values:
+    nodejs:
+      disableTraefikTls: false
+      image: hmctspublic.azurecr.io/bar/web:prod-9dcca15-20220808172206 #{"$imagepolicy": "flux-system:bar-web"}
```
Comment (thomast1906) [inline k8s/namespaces/bar/bar-web-int/demo.yaml:10]: Does image not need updated with new imagePolicy being added?

**Review by anoojhmcts — DISMISSED** [review]

**Review by hannah38 — APPROVED** [review]

**Pull request merged**